conf/defaults.ini

```diff
@@ -235,7 +235,7 @@ admin_password = admin
 secret_key = SW2YcwTIb9zpOOhoPsMm
 
 # current key provider used for envelope encryption, default to static value specified by secret_key
-encryption_provider = secretKey
+encryption_provider = secretKey.v1
```

Comment:
…because the old value for this field should still remain valid, if it has been present in a release.